cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/network/protocol/chunk/ChunkedPacketFactory.java

```diff
@@ -0,0 +1,53 @@
+package de.dytanic.cloudnet.driver.network.protocol.chunk;
+
+import de.dytanic.cloudnet.common.document.gson.JsonDocument;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+public class ChunkedPacketFactory {
```

Comment:
Codacy found an issue: [The utility class name 'ChunkedPacketFactory' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6005504)